### DISCUSSION TITLE
There is a way to make a temple to adore the 5 gods same time?

### DISCUSSION THREAD

**Discussion by BraianCAESAR** [Keriew/augustus#269] [Ideas]
Its boring construicting 5 temples all the time, would be cool if we could construct only one temple that works for all gods. Even if this temple uses 10 peoples to work.

**Comment by crudelios** [comment]
You can do that already by building oracles!

**Reply by BraianCAESAR** [reply]
But Oracles have not priests. 1 single big temple could creat 5 priests walking around the city

**Reply by crudelios** [reply]
Ah, then no,  I don't think we should change such a fundamental game mechanic.
